sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -36,5 +37,13 @@ def finish_transaction(transaction, status)
         transaction.finish
       end
     end
+
+    class SentryContextClientMiddleware
+      def call(_worker_class, job, _queue, _redis_pool)
```

Comment:
Should we call the following methods? I'm not sure.

- `return yield unless Sentry.initialized?`
- Sentry.clone_hub_to_current_thread

Reply:
`return yield unless Sentry.initialized?` is needed but `Sentry.clone_hub_to_current_thread` is not.

Reply:
I'll fix, thanks :+1: